docs/content/stable/api/ysql/exprs/func_yb_hash_code.md

```diff
@@ -0,0 +1,195 @@
+---
+title: yb_hash_code() function [YSQL]
```

Comment:
ditto about the comments -- make sure once you land you verify the diff between latest and stable versions are only the expected macro stuff (e.g. the path being stable vs latest).